TypeCobol/Compiler/Scanner/Scanner.cs

```diff
@@ -267,6 +274,8 @@ public static void ScanTokensLineContinuationGroup(IList<TokensLine> continuatio
                     break;
                 }
             }
+            // Positions in the concatainedLine string where multi strings must be applied.
```

Comment:
Typo : concatained does not exist. 'concatenated' is correct.

Reply:
Sorry about typo: 1dc0d56